templates/jtimon.json.j2

```diff
@@ -1,7 +1,9 @@
 {
     "config_file_list": [
+{% if jtimon_config_hosts | length > 0 %}
```

Comment:
Do we need this? Normally the loop shouldn't run if `length == 0`

Reply:
true, not needed